_saas-integrations/recharge/v1/recharge-v1.md

```diff
@@ -0,0 +1,133 @@
+#         Tap Details        #
+# -------------------------- #
+
+name: "recharge"
+display_name: "ReCharge"
+
+singer: true
+status-url: ""
+
+tap-name: "recharge"
+repo-url: https://github.com/singer-io/tap-recharge
+
+this-version: "1.0"
+
+api: |
+  [ReCharge Payments API](https://developer.rechargepayments.com/){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. To access the {{ integration.display_name }} API documentation, you will need [your API key](#obtain-api-key). Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**A Shopify Account**. You need this to access your {{ integration.display_name }} information."
+
+setup-steps:
+  - title: "Obtain Your {{ integration.display_name }} API Key"
+    anchor: "obtain-api-key"
+    content: |
+      1. Log into your Shopify account, select `Apps`, and then click on your {{ integration.display_name }} application.
+         {% include layout/image.html file="/integrations/recharge-apps-screen.png" alt="Shopify Apps page." enlarge=true max-width="550" %}
+      {:start="2"}
+      2. In the application, click on the `EXPLORE {{ integration.display_name }}` option.
+         {% include layout/image.html file="/integrations/recharge-explore-button.png" alt="EXPLORE ReCharge button." enlarge=true max-width="250" %}
+      {:start="3"}
+      3. Click on the `Integrations` tab, and then click `API tokens...`.
+         {% include layout/image.html file="/integrations/recharge-integrations-api-tokens.png" alt="Integrations page with API token link." enlarge=true max-width="550" %}
+      {:start="4"}   
+      4. Click on `Create an API token` and fill out the `Details` and `Permission`. In the `Permission` section, select `Read Access` from the dropdown menu for each item. Save to obtain your API key and keep it readily available to set up the inegration.
```

Comment:
- **Bold** instead of `code`
- I'd break this down into at least two more steps - maybe:

   ```
   4. Click **Create API token**.
   5. On the **Create an API token** page:
      - In the **Details** section, enter a **Nickname** and **Contact email** into their respective fields.
      - In the **Permission** section, select the `Read Access` scope for each permission.
   6. Click **Save** to create the API token.
```